sphinx_thebe/__init__.py

```diff
@@ -75,41 +61,10 @@ def update_thebe_context(app, doctree, docname):
         else:
             kernel_name = "python3"
 
-    # Codemirror syntax
-    cm_language = kernel_name
-    if "python" in cm_language:
-        cm_language = "python"
-    elif cm_language == "ir":
-        cm_language = "r"
-
-    # Create the URL for the kernel request
-    repo_url = config_thebe.get(
-        "repository_url",
-        "https://github.com/binder-examples/jupyter-stacks-datascience",
-    )
-    branch = config_thebe.get("repository_branch", "master")
-    path_to_docs = config_thebe.get("path_to_docs", ".").strip("/") + "/"
-    org, repo = _split_repo_url(repo_url)
-
     # Update the doctree with some nodes for the thebe configuration
     thebe_html_config = f"""
     <script type="text/x-thebe-config">
-    {{
-        requestKernel: true,
-        binderOptions: {{
-            repo: "{org}/{repo}",
-            ref: "{branch}",
-        }},
-        codeMirrorConfig: {{
-            theme: "{codemirror_theme}",
-            mode: "{cm_language}"
-        }},
-        kernelOptions: {{
-            kernelName: "{kernel_name}",
-            path: "{path_to_docs}{str(Path(docname).parent)}"
-        }},
-        predefinedOutput: true
-    }}
+    { json.dumps(thebe_config) }
```

Comment:
Is there any reason we'd wanna do any data validation or anything?

Reply:
I think validation is a great idea, however the best approach to implement validation would be https://github.com/executablebooks/thebe/issues/398.